app/Main.hs

```diff
@@ -45,14 +85,17 @@ main = do
 
       headState :: HeadState
       headState = HeadState userCreds networkId pparamsResult
+  putStrLn $ "user pubKeyHash " <> show userCreds.userPubKeyHash
+  Text.putStrLn $ "user address " <> serialiseAddress userCreds.userAddress
```

Comment:
nit: you should be able to add here if you want a trailing newline `<> "\n"`

Reply:
Of course, thanks! Especially since I'm already do exactly this 10 lines later :)